manifests/node_exporter.pp

```diff
@@ -113,19 +124,28 @@
   validate_bool($manage_user)
```

Comment:
please drop those validate_* calls. They are not needed if we have datatypes.

Reply:
done